community/lucene-index/src/main/java/org/neo4j/kernel/api/impl/labelscan/LuceneLabelScanStore.java

```diff
@@ -114,9 +116,22 @@ else if ( !luceneIndex.isValid() )
         }
     }
 
+    private void create() throws IOException
```

Comment:
It'll just be a volatile reference somewhere else, won't it?

Reading a volatile which no one ever writes to, except ones at startup, aint so bad (knowing details about the java memory model), is it?